### ISSUE TITLE
simple way to use gltf/glb as input mesh? 

### ISSUE THREAD

**Issue by arpu** [Simon-Steinmann/collision_mesh_optimization#1]
Hello

would it be possible to use gltf/glb as input Mesh?

**Comment by Simon-Steinmann** [comment]
I dont think so. You could try transforming it into a .stl, .dae or .obj file first. I'm sure that there are converters online for that.

**Comment by arpu** [comment]
Hey, make sense will try Blender to convert to .obj first
